packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -711,8 +711,12 @@ mixin WidgetInspectorService {
   @protected
   static set instance(WidgetInspectorService instance) {
     _instance = instance;
+
   }
 
+  /// Information about the service protocol for the running application.
+  developer.ServiceProtocolInfo? serviceInfo;
```

Comment:
Does this have to be public? If so, the doc is a little sparse...

Reply:
made private and added more details to doc